public/app/features/dashboard-scene/scene/layout-tabs/TabsLayoutManager.tsx

```diff
@@ -78,32 +113,33 @@ export class TabsLayoutManager extends SceneObjectBase<TabsLayoutManagerState> i
     this.state.tabs.forEach((tab) => tab.getLayout().activateRepeaters?.());
   }
 
-  public removeTab(tab: TabItem) {
-    if (this.state.currentTab === tab) {
-      const currentTabIndex = this.state.tabs.indexOf(tab);
-      const nextTabIndex = currentTabIndex === 0 ? 1 : currentTabIndex - 1;
-      const nextTab = this.state.tabs[nextTabIndex];
-      this.setState({ tabs: this.state.tabs.filter((t) => t !== tab), currentTab: nextTab });
+  public removeTab(tabToRemove: TabItem) {
+    // Do not allow removing last tab (for now)
```

Comment:
(nit for one particular edge case) The logic for allowing last tab removal is in case one would want to start from scratch without having to remove every panel within that tab one by one. A shorthand for: add new tab and then remove the existing one.

Reply:
@bfmatei yea, but it crashes (in main) so added this to prevent the crash for now :)